src/server/rest/v1/service/AuthorizationService.ts

```diff
@@ -499,6 +507,28 @@ export default class AuthorizationService {
     return userSites.result.map((userSite) => userSite.siteID);
   }
 
+  private static async getSiteOwnerSiteIDs(tenantID: string, userToken: UserToken): Promise<string[]> {
+    // Get the Sites where the user is Site Owner
+    const userSites = await UserStorage.getUserSites(tenantID,
+      {
+        userID: userToken.id,
+        siteOwner: true
+      }, Constants.DB_PARAMS_MAX_LIMIT,
+      ['siteID']
+    );
+    return userSites.result.map((userSite) => userSite.siteID);
+  }
+
+  private static async getAssignedUserSites(tenantID: string, userID: string) {
```

Comment:
Missing return type

Reply:
I just realised this was not actually needed. Commit with refactored code coming soon.